Directory.Packages.props

```diff
@@ -3,25 +3,25 @@
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
   </PropertyGroup>
   <ItemGroup>
-    <PackageVersion Include="Fabulous" Version="3.0.0-pre9" />
+    <PackageVersion Include="Avalonia.iOS" Version="11.2.1" />
+    <PackageVersion Include="Fabulous" Version="3.0.0-pre16" />
```

Comment:
@edgarfgp Just tried upgrading to `Fabulous.Avalonia` 3.0.0-pre7, but can't compile because 3.0.0-pre9 is the latest publicly available package version for `Fabulous` at the moment.

Reply:
My bad - manually editing the Version worked. I was fooled by the VS GUI not offering me that version to upgrade to for some reason. Sorry for the false alarm and thanks for the heads up!